### ISSUE TITLE
Save/load checkpoints during model training

### ISSUE THREAD

**Issue by ignaciorlando** [ignaciorlando/cnn-dr-kaggle#12]
Based on [this example](https://keras.io/callbacks/#example-model-checkpoints), save/load intermediate models using callbacks.

**Comment by ignaciorlando** [comment]
It is not that easy. See [Keras #1872.](https://github.com/fchollet/keras/issues/1872).

Maybe we can try doing something like this:
1. Write a History class that updates a log file with current epoch, learning rate and additional information on_epoch_end.
2. Use model.load_weights() to initialize the weights of the CNN.
3. Assign the hyperparameters according to the log file (including the epoch).
